src/amo/components/LanguageTools/index.js

```diff
@@ -32,6 +32,19 @@ type LanguageToolListProps = {|
   addons?: Array<AddonType>,
 |};
 
+
+// Get languages into a list of objects sorted by the english name.
+const languageList = Object.keys(languages).map((langKey) => {
+  return {
+    english: languages[langKey].English,
+    locale: langKey,
+    native: languages[langKey].native,
+  };
+}).sort((a, b) => {
```

Comment:
Here it should be:

``` js
.sort((languageA: Language, languageB: Language): bool => {
   return languageA.localeCompare(languageB);
});
```

Reply:
I wasn't too sure about using this, but then I see we already are: https://github.com/mozilla/addons-frontend/blob/cfcd838c812b1d0f2aa84c47c498f971e4a417d0/src/core/reducers/categories.js#L68